.github/workflows/auto_cherry_pick.yml

```diff
@@ -43,3 +43,32 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.to_branch }}
+
+  create-issue:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
+    needs: auto-cherry-pick
+    steps:
+      - name: Create Issue on Failed workflow
+        uses: dacbd/create-issue-action@main
+        with:
+          token: ${{ github.token }}
+          title: "[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}"
+          body: |
+            #### Auto-Cherry-Pick WorkFlow Failure:
+            - [Failed Cherrypick Action](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})
+            - [Parent Pull Request](https://github.com/${{ github.repository }}/pull/${{ github.event.number }})
+          labels: Failed_AutoCherryPick
+          assignees: "${{ github.event.pull_request.user.login }}"
+
+  google-chat-notification:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') }}
+    needs: auto-cherry-pick
```

Comment:
Flip !

Reply:
Most of the examples have seen GitHub docs use the `always(), failure(), cancelled()` expression before doing any further logical  addition